include/alpaka/pltf/PltfCpu.hpp

```diff
@@ -21,7 +21,8 @@ namespace alpaka
     {
         //#############################################################################
         //! The CPU device platform.
-        class PltfCpu
+        class PltfCpu :
+            public concepts::Implements<ConceptPltf, void>
```

Comment:
Can you please use the type of the current class `PltfCpu` instead of void? This allows to use the rest of the concept base type logic later. I would also like to add a `static_assert` into `Implements` that this parameter is not `void`.

Reply:
OK will use the class itself e.g. `public concepts::Implements<ConceptPltf, PltfCpu>`